qt/aqt/mpv.py

```diff
@@ -232,7 +232,7 @@ def _reader(self):
         while not self._stop_event.is_set():
             if isWin:
                 try:
-                    (n, b) = win32file.ReadFile(self._sock, 4096)
+                    (n, b) = win32file.ReadFile(self._sock, 4096)[1]
```

Comment:
This looks incorrect

Reply:
I first removed the n, then while re reading I thought that it's more clear to leave it this way, even if `n` is never used, so I added it back. Not correctly, sorry